sssd_test_framework/roles/client.py

```diff
@@ -97,6 +98,16 @@ def setup(self) -> None:
 
                 self.sssd.import_domain(domain, role)
 
+    def infopipe(self) -> DBUSDestination:
```

Comment:
There is no need to have this as a method and no reason to postpone creation of the object since the constructor does not do anything difficult. Can you rather move it into `__init__`? And maybe name it `ifp` to make it shorter?

Reply:
> The constructor doesn't do anything complex, as you said, but I wanted to avoid connecting to the bus if infopipe will not be used (most of the non-dbus tests). What do you think?

But it currently doesn't connect in constructor, or does it? We call dbus-send only when we run introspection via `getObject`?